app/models/pxe_image_type.rb

```diff
@@ -4,7 +4,7 @@ class PxeImageType < ApplicationRecord
   has_many :windows_images
   has_many :iso_images
 
-  validates_uniqueness_of :name
+  validates_uniqueness_of :name, :unique_within_region => true, :message => "Pxe Image Type name (%{value}) has already been taken"
```

Comment:
Was there a problem with the default message?

Reply:
Absolutely, it doesn't distinguish between the pxe_image_type name and the name of the customization_template.

Reply:
But those are different screens
![image](https://user-images.githubusercontent.com/651659/35538293-3d67108c-051b-11e8-91c7-6d0cf4022de5.png)
![image](https://user-images.githubusercontent.com/651659/35538342-6dd187ac-051b-11e8-8e0a-2b2e3d5f5503.png)